### PR TITLE
Added Support for Label Releases endpoint

### PR DESCRIPTION
**Pull request by maggiolo00**
Hi @afonso360 
me again :D

I've done the same work for label releases.

### CODE REVIEW

File: src/data_structures/label.rs [hunk 1]
```diff
@@ -140,6 +146,44 @@ impl LabelQueryBuilder {
...
+
+        let result: Result<String, QueryError> = self.perform_request();
+
+        if let Err(error) = result {
+            return Err(error);
+        } else {
+            let result_string = result.ok().unwrap();
+            let json = serde_json::from_str(&result_string);
+
+            if let Ok(artist) = json {
```
Comment (afonso360) [inline src/data_structures/label.rs]: I know its a minor thing, could you please change this to label?

Comment (maggiolo00) [inline src/data_structures/label.rs]: Sure

**Comment by maggiolo00** [comment]
Done

i did tag the example on the function with

rust,no_run

I saw that the documentation tests run and they do requests to discogs. In this way the test only does the compilation test

**Comment by afonso360** [comment]
Yeah, thats good, we should probably do that for the other documentation tests that actually do requests, the way im planning on doing this is, having all the tests that performe requests on the separate tests folder.

Again, thank you for your PR

**Pull request merged**